reports/2017-03-19.md

```diff
@@ -37,7 +37,7 @@ Welcome everyone, looking forward to your contrinutions and working with you. Pl
 
 ## This week
 
-We are working towards a simpler and better documented basic flow. Early next week the `moby` tool will be changed to have `moby build` and `moby run` subcommands. At least initially the `moby run` flow will only be targeted at running locally (hyperkit and kvm, qemu). We may add a third cluster run option later, using infrakit. @rneugeba has been working on Go hyperkit bindings to make this easier in [#1327](https://github.com/docker/moby/pull/1327).
+We are working towards a simpler and better documented basic flow. Early next week the `moby` tool will be changed to have `linuxkit build` and `moby run` subcommands. At least initially the `moby run` flow will only be targeted at running locally (hyperkit and kvm, qemu). We may add a third cluster run option later, using infrakit. @rneugeba has been working on Go hyperkit bindings to make this easier in [#1327](https://github.com/docker/moby/pull/1327).
```

Comment:
we can probably revert all changes to `reports/` since it was correctly specified at its point of writing